decidim/decidim
# Fix single initiative type

#### :tophat: What? Why?
When there is only one initiative, it is impossible to edit the initiative because the form needs a type_id to generate scope id and sample signatures.

This PR fixes this problem.

#### :pushpin: Related Issues
- Related to #5835

#### Testing
* Delete initiatives types to keep only one.
* Create initiative in remaining type
* Publish then edit it
* See there is no error

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*

:hearts: Thank you!
